src/context/fetch.ts

```diff
@@ -42,11 +57,9 @@ export const fetch = <ResponseType = any>(
     )
   }
 
-  const { body } = input
-  const compliantReq: RequestInit = augmentRequestInit({
-    ...input,
-    body: typeof body === 'object' ? JSON.stringify(body) : body,
-  })
+  const requestParameters: RequestInit = createFetchRequestParameters(input)
+
+  const compliantReq: RequestInit = augmentRequestInit(requestParameters)
```

Comment:
We should pick one: either `request` or `req`. Since we are already using the longer version in the codebase, I suggest to name this `compliantRequest`. Nothing wrong with being a little more specific :)